packages/world-module-erc20/src/ERC20Pausable.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MIT
+// Adapted from OpenZeppelin's ERC20Pausable extension
```

Comment:
Should we add a gitub permalink to the OZ files for all of these? Might make it easier in the future to compare, especially if OZ releases new contract versions

Reply:
will do 🫡